tests/integration/plugins/test_jlink.py

```diff
@@ -89,6 +89,40 @@ def test_jlink_plugin_embedded_jar(new_dir, partitions):
     assert len(list(Path(f"{new_dir}/stage/usr/lib/jvm/").rglob("libawt.so"))) > 0
 
 
+@pytest.mark.usefixtures("build_test_jar")
+def test_jlink_plugin_add_modules(new_dir, partitions):
+    parts_yaml = textwrap.dedent(
+        """
+        parts:
+            my-part:
+                plugin: jlink
+                source: .
+                jlink-jars: ["test.jar"]
+                jlink-add-modules: ["jdk.crypto.ec", "java.sql"]
```

Comment:
What is the difference between a module and a package? The internet tells me that a module is a collection of packages, is that it?
For "java.sql" I see that apparently there's both a module and a package: https://docs.oracle.com/en/java/javase/11/docs/api/java.sql/module-summary.html . Does `jlink-add-modules` only apply to modules?